pkg/machines/actions/provider-actions.es6

```diff
@@ -59,8 +59,8 @@ import {
  *  The naming convention for action creator names is: <verb><Noun>
  *  with the present tense.
  */
-export function attachDisk({ connectionName, diskFileName, target, permanent, hotplug, vmName }) {
-    return virt(ATTACH_DISK, { connectionName, diskFileName, target, permanent, hotplug, vmName });
+export function attachDisk({ connectionName, diskFileName, target, permanent, hotplug, vmName, vmId }) {
+    return virt(ATTACH_DISK, { connectionName, diskFileName, target, permanent, hotplug, vmName, vmId });
```

Comment:
It might make sense to split out the "vmID" changes to a separate PR that can already land. That's the kind of preparatory PR that is rather invasive (thus bitrots fast) but also doesn't/shouldn't change functionality much, so you can land it fast.

Reply:
Done